health/health.h

```diff
@@ -4,6 +4,7 @@
 #define NETDATA_HEALTH_H 1
 
 #include "../daemon/common.h"
+#include "../aclk/agent_cloud_link.h"
```

Comment:
does this belong into this PR?

Reply:
This gives a compile error if removed - I will keep it

Reply:
oh... my mistake in this case

Reply:
This is related to the other comment where we shouldn't compile the whole code. By removing the compilation of the `aclk` directory this problem goes away, by hiding this `include` with ` #ifdef ACLK_ENABLE`.

Moreover, this line should be moved inside `daemon/common.h` near the line:
```
// netdata agent claiming
#include "claim/claim.h"
```